### DISCUSSION TITLE
Welcome to DALLE2-pytorch Discussions!

### DISCUSSION THREAD

**Discussion by lucidrains** [lucidrains/DALLE2-pytorch#6] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MrPalais** [comment]
to install (on windows) : 
conda install -c conda-forge youtokentome 
then the pip install works... BUT it seems that there is something missing in cli.py because "image" is not defined :(

Traceback (most recent call last):
  File "C:\conda\envs\da\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\conda\envs\da\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\conda\envs\da\Scripts\dream.exe\__main__.py", line 7, in <module>
  File "C:\conda\envs\da\lib\site-packages\click\core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "C:\conda\envs\da\lib\site-packages\click\core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "C:\conda\envs\da\lib\site-packages\click\core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "C:\conda\envs\da\lib\site-packages\click\core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "C:\conda\envs\da\lib\site-packages\dalle2_pytorch\cli.py", line 9, in dream
    return image
NameError: name 'image' is not defined

**Reply by ugurcansakizli** [reply]
This is because the code is not ready yet, which is indicated in the code as (wip) and (work in progress) and (the code is not working at the moment, please be patient and if you can / want to help please join discord).

**Comment by Atha150** [comment]
I let the programm run, it makes iterations, but where is the image? where do i find it?

**Reply by lucidrains** [reply]
patience, it is not trained yet. i'm exercising all my knowledge and engineering expertise to make it happen

**Reply by lucidrains** [reply]
you can help make this happen sooner by participating too, by joining the Laion discord!

**Comment by HEGADE** [comment]
hey hi ,how can save the image  generated from the  images=dlle2(text)
![image](https://user-images.githubusercontent.com/60566946/219371584-a0ee4019-d72c-45d0-a7ef-bb514c81e6d2.png)
